README.md

```diff
@@ -15,9 +15,10 @@
 
 # Database creation
   
-  docker-compose exec web rails db:create
+  docker-compose run web rake db:create
```

Comment:
This change is not described in the commit message

Reply:
Thanks for the comment, we will take it into account. commit massage "Change database creation and migration commands"